src/app/modules/map-config/services/map-layer-form-builder/map-layer-form-builder.service.ts

```diff
@@ -530,12 +516,7 @@ export class MapLayerTypeClusterFormGroup extends MapLayerAllTypesFormGroup {
           marker('Granularity'),
           marker('Granularity description'),
           false,
-          [
-            { label: marker('Coarse'), value: Granularity.coarse },
-            { label: marker('Medium'), value: Granularity.medium },
-            { label: marker('Fine'), value: Granularity.fine },
-            { label: marker('Finest'), value: Granularity.finest }
-          ]
+          Object.keys(Granularity).map(key => { return { label: marker(Granularity[key].toString()), value: Granularity[key] } })
```

Comment:
`marker(..)` is not necessary because it relies on text and here it's a variable

Reply:
fixed